webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/DataSourceService.scala

```diff
@@ -23,10 +23,16 @@ import play.api.inject.ApplicationLifecycle
 import play.api.libs.json.Json
 import org.joda.time.format.ISODateTimeFormat
 
+import scala.collection.mutable
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
 import scala.io.Source
 
+case class ResumableUploadInformation(chunkSize: Int, totalChunkNumber: Long)
```

Comment:
consider renaming totalChunkNumber to totalChunkCount to distinguish it from the currentChunkNumber